File: app/src/main/java/com/example/ikastyle/Dao/HeadGearDao.java
package com.example.ikastyle.Dao;

import androidx.room.Dao;
import androidx.room.Query;

import com.example.ikastyle.Entity.HeadGear;

import java.util.List;

@Dao
public interface HeadGearDao {
    @Query("SELECT * FROM MAST_HEAD_GEAR WHERE language_code = :languageCode ORDER BY name")
    List<HeadGear> getHeadGearList(int languageCode);
}

File: app/src/main/java/com/splatool/ikastyle/model/data/dao/HeadGearDao.kt
package com.splatool.ikastyle.model.data.dao

import com.splatool.ikastyle.model.data.entity.HeadGear
import androidx.room.*

@Dao
interface HeadGearDao {
    @Query("SELECT * FROM MAST_HEAD_GEAR WHERE language_code = :languageCode ORDER BY name")
    suspend fun getHeadGearList(languageCode: Int): List<HeadGear>
}